src/utils/xcmApi/index.ts

```diff
@@ -0,0 +1,78 @@
+import {
+  createSubstrateAPI,
+  CollateralUnit,
+  DefaultTransactionAPI
+} from '@interlay/interbtc-api';
+import {
+  Currency,
+  MonetaryAmount
+} from '@interlay/monetary-js';
+import { web3FromAddress } from '@polkadot/extension-dapp';
+import { AddressOrPair } from '@polkadot/api/types';
+
+import {
+  RELAYCHAIN_URL,
+  RELAYCHAIN_ID
+} from '../../constants';
+
+const xcmTransfer = async (
+  // Pass this in explicitly to avoid creating
+  // a dependency on the interBCT api instance
+  originatingAccount: AddressOrPair,
+  destinationAddress: string,
+  transferAmount: MonetaryAmount<Currency<CollateralUnit>, CollateralUnit>
```

Comment:
Is it just `CollateralUnit` or other currencies too?

Reply:
Just the collateral unit for the MVP (transferring KSM only).